src/lib389/lib389/backend.py

```diff
@@ -511,22 +511,23 @@ def _lint_mappingtree(self):
 
     def _lint_cl_trimming(self):
         """Check that cl trimming is at least defined to prevent unbounded growth"""
+        bename = self.lint_uid()
         suffix = self.get_attr_val_utf8('nsslapd-suffix')
         replicas = Replicas(self._instance)
-        replica = replicas.get(suffix)
-        bename = self.lint_uid()
-        if replica is not None:
+        try:
+            # Check if replication is enabled
+            replicas.get(suffix)
+            # Check the changelog
             cl = Changelog(self._instance, suffix=suffix)
-            try:
-                if cl.get_attr_val_utf8('nsslapd-changelogmaxentries') is None and \
-                   cl.get_attr_val_utf8('nsslapd-changelogmaxage') is None:
-                    report = copy.deepcopy(DSCLLE0001)
-                    report['fix'] = report['fix'].replace('YOUR_INSTANCE', self._instance.serverid)
-                    report['check'] = f'backends:{bename}::cl_trimming'
-                    yield report
-            except:
-                # No changelog
-                pass
+            if cl.get_attr_val_utf8('nsslapd-changelogmaxentries') is None and \
+               cl.get_attr_val_utf8('nsslapd-changelogmaxage') is None:
+                report = copy.deepcopy(DSCLLE0001)
+                report['fix'] = report['fix'].replace('YOUR_INSTANCE', self._instance.serverid)
+                report['check'] = f'backends:{bename}::cl_trimming'
+                yield report
+        except:
+            # Suffix is not replicated
```

Comment:
Might just be worth a log.debug to say "skipping bename as no replication enabled"  so that if we have a CU issue and we get debug logs we can see there may be a replication agreement inbalance?

Reply:
Done